# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rocjpeg/hotfix-cc.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rocJPEG-rocm-6.3.3/CMakeLists.txt.vanilla	2025-04-03 17:18:35.525353931 +0200
+++ rocJPEG-rocm-6.3.3/CMakeLists.txt	2025-04-03 17:18:52.275650695 +0200
@@ -74,7 +74,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH} ${ROCM_PATH}/hip)
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 # rocJPEG Default Options
 option(BUILD_WITH_AMD_ADVANCE "Build rocJPEG for advanced AMD GPU Architecture" OFF)
--- rocJPEG-rocm-6.3.3/samples/jpegDecodeMultiThreads/CMakeLists.txt.vanilla	2025-04-03 17:18:35.515776902 +0200
+++ rocJPEG-rocm-6.3.3/samples/jpegDecodeMultiThreads/CMakeLists.txt	2025-04-03 17:18:41.851799085 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 find_package(HIP QUIET)
 
@@ -80,4 +80,4 @@
     if (NOT Threads_FOUND)
         message(FATAL_ERROR "-- ERROR!: Threads Not Found! - please insatll Threads!")
     endif()
-endif()
\ No newline at end of file
+endif()
--- rocJPEG-rocm-6.3.3/samples/jpegDecodeBatched/CMakeLists.txt.vanilla	2025-04-03 17:18:35.519021050 +0200
+++ rocJPEG-rocm-6.3.3/samples/jpegDecodeBatched/CMakeLists.txt	2025-04-03 17:18:44.637000000 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 find_package(HIP QUIET)
 
@@ -71,4 +71,4 @@
     if (NOT ROCJPEG_FOUND)
         message(FATAL_ERROR "-- ERROR!: rocJPEG Not Found! - please install rocJPEG!")
     endif()
-endif()
\ No newline at end of file
+endif()
--- rocJPEG-rocm-6.3.3/samples/jpegDecode/CMakeLists.txt.vanilla	2025-04-03 17:18:35.522060612 +0200
+++ rocJPEG-rocm-6.3.3/samples/jpegDecode/CMakeLists.txt	2025-04-03 17:18:47.813293720 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 find_package(HIP QUIET)
 
@@ -71,4 +71,4 @@
     if (NOT ROCJPEG_FOUND)
         message(FATAL_ERROR "-- ERROR!: rocJPEG Not Found! - please install rocJPEG!")
     endif()
-endif()
\ No newline at end of file
+endif()
